packages/dx-react-scheduler/docs/guides/current-time-indication.md

```diff
@@ -0,0 +1,22 @@
+# React Scheduler - Current Time Indication
+
+The React Scheduler can display a current time indicator and shade appointments and cells up to the current time. This feature can be enabled in the day, week, and month views, but the month view supports only the shading.
+
+## Related Plugins
+
+The following plugin implements this feature:
```

Comment:
Why write an introduction if there is only one plugin?

Reply:
```suggestion
The [CurrentTimeIndicator](../reference/current-time-indicator.md) plugin renders the current time indicator and shades that cover appointments and timetable cells up to the current time. 
```

Reply:
and the description also duplicates information you already provided.

Reply:
This is just what we do [in all articles](https://devexpress.github.io/devextreme-reactive/react/scheduler/docs/guides/appointment-tooltip/). I guess we can remove the introduction because the title of this section is explanatory enough, but I'm against giving the info as a sentence because that would break the pattern.

Reply:
Well, it seem strange to add an introductory sentence when you only have one item and one sentence. That is like saying 
"I'm going to the bank at the following time: 
- 16:00"

Reply:
Agreed.
```suggestion

```